presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -61,4 +62,16 @@ public BaseJdbcConfig setConnectionPassword(String connectionPassword)
         this.connectionPassword = connectionPassword;
         return this;
     }
+
+    public boolean isMapLowercaseTableNames()
+    {
+        return mapLowercaseTableNames;
+    }
+
+    @Config("connection-map-tables")
+    public BaseJdbcConfig setMapLowercaseTableNames(boolean mapLowercaseTableNames)
```

Comment:
I don't think this should be configurable as the feature is needed in order for the connectors to work properly given Presto's limitations on lowercase table names.